### PR TITLE
Continuation on Ethereum payments

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by badb** [comment]
TestEthereumTransactionSystem.test_init fails - can't remove test dir - chaindata is used. Same problem with EthereuClientTest.test_client.

**Comment by chfast** [comment]
Thanks. I will take care of that.

**Comment by chfast** [comment]
Blocked by https://github.com/imapp-pl/golem/pull/279.

**Comment by chfast** [comment]
@badb Can you test again on non-virtualized Windows?

File: golem/transactions/ethereum/paymentprocessor.py [hunk 1]
```diff
@@ -58,28 +59,45 @@ def __init__(self, client, privkey):
         # TODO: Maybe it should not be the part of this class
         # TODO: Allow seting timeout
         # TODO: Defer a call only if payments waiting
-        scheduler = LoopingCall(lambda: self.sendout())
+        scheduler = LoopingCall(lambda: self.run())
```
Comment (badb) [inline golem/transactions/ethereum/paymentprocessor.py]: LoopingCall(self.run) is not working?

**Comment by badb** [comment]
Seems to be working :+1:

File: golem/transactions/ethereum/paymentprocessor.py (hunk 1 above)
Comment (chfast) [inline golem/transactions/ethereum/paymentprocessor.py]: Probably is. I always forget that in Python function is a value.

**Pull request merged**